src/expenses/expenses.controller.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+const expenseController = require('./expenses.services');
+const userController = require('../users/users.services');
+
+const expenseKeys = [
+  'userId',
+  'spentAt',
+  'title',
+  'amount',
+  'category',
+];
+
+const checkProps = (expense) => expenseKeys
+  .every(prop => expense.hasOwnProperty(prop));
+
+const getAll = (req, res) => {
+  res.send(expenseController.getAll(req.query));
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+  const expense = expenseController.getByID(+id);
+
+  if (!expense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(expense);
```

Comment:
Do not forget about the successful response code
```suggestion
  res.status(200).send(expense);
```